examples/pxScene2d/src/pxScene2d.h

```diff
@@ -153,6 +155,7 @@ class rtFileDownloadRequest;
 class pxScene2d;
 class pxScriptView;
 class pxFontManager;
+#if 0
```

Comment:
Let's remove this code from pxScene2d.h instead of #if 0.

Reply:
Of course ... thanks !